### PR TITLE
Syrytsia viktor

### PR DESCRIPTION
**Pull request by ViktorSyrytsia**

### CODE REVIEW

File: src/components/app/App.js [hunk 1]
```diff
@@ -1,24 +1,34 @@
...
-      <Header />
-      <div>content will appear here soon</div>
-      {/*  todo: здесь нужно использовать Footer по аналогии с Header */}
+    return (
+        <div className="App">
+            <Header/>
+            <div>content will appear here soon</div>
+            <Footer/>
+            <div className="card-group">
+                {usersList.map( user => (
```
Comment (AlyonaPianykh) [inline src/components/app/App.js:22]: Very nice! And remove space after `(`

File: src/components/footer/Footer.js [hunk 2]
```diff
@@ -1,15 +1,33 @@
...
-  //   в котором будет 2 div блока: 1й с классом "may-footer-content", второй будет без класса но с текстом 'All rights reserved. may-app © 2020'
-  //   в первом div блоке должны быть вложены еще 2 блока div: 1й с текстом "Some text about authors"
-  //   второй будет иметь класс "may-footer-social-media-wrapper", в него будут вложены 3 блока img в соответствии с socialMediaIcons
-  //   каждый img должен иметь класс "may-footer-social-media-icon" и в пропсу src должен быть записан src из соответствующего объекта в socialMediaIcons массиве
-  //   и в пропсу alt должен быть записан alt из соответствующего объекта в socialMediaIcons массиве
+                </div>
+            </div>
+            <div>
+                {rights}
+            </div>
```
Comment (AlyonaPianykh) [inline src/components/footer/Footer.js:27]: This empty line should be deleted

File: src/components/footer/Footer.js [hunk 3]
```diff
@@ -1,15 +1,33 @@
...
 
+        </div>
+    )
+
```
Comment (AlyonaPianykh) [inline src/components/footer/Footer.js:30]: And this one too

File: src/components/user-card/UserCard.js [hunk 4]
```diff
@@ -1,15 +1,52 @@
...
+            </div>
+
+        </div>
+    )
+
+}
+
+export default UserCard;
+
+
```
Comment (AlyonaPianykh) [inline src/components/user-card/UserCard.js:47]: These empty lines should be deleted

**Comment by AlyonaPianykh** [comment]
Good job! Ping me in telegram I will give you extra task

**Comment by ViktorSyrytsia** [comment]
closed

**Pull request closed without merging**